src/web/components/MainMedia.tsx

```diff
@@ -2,11 +2,14 @@ import React from 'react';
 import { css, cx } from 'emotion';
 
 import { until } from '@guardian/src-foundations/mq';
+import { Display } from '@guardian/types/Format';
 
 import { ImageComponent } from '@root/src/web/components/elements/ImageComponent';
 import { YoutubeBlockComponent } from '@root/src/web/components/elements/YoutubeBlockComponent';
+import { YoutubeEmbedBlockComponent } from '@root/src/web/components/elements/YoutubeEmbedBlockComponent';
 import { EmbedBlockComponent } from '@root/src/web/components/elements/EmbedBlockComponent';
-import { Display } from '@guardian/types/Format';
```

Comment:
Was `Display` unused here? Or have you removed something that was using it?

Reply:
no I just moved it a couple lines (see line 5), I like to seperate imports from external packages from internal imports

Reply:
ah yeah. I see it now. I prefer that too.